beacon_node/beacon_chain/src/chain_config.rs

```diff
@@ -67,6 +67,8 @@ pub struct ChainConfig {
     pub prepare_payload_lookahead: Duration,
     /// Use EL-free optimistic sync for the finalized part of the chain.
     pub optimistic_finalized_sync: bool,
+    /// Whether backfill sync rate-limiting is disabled.
+    pub disable_backfill_rate_limiting: bool,
```

Comment:
Agree with you from the code readability perspective. From a user experience perspective though, I think it's more user friendly when we name a boolean parameter according to what the default value is - since users should only use the parameter if they want to change the default behaviour - and the initial default would drive the naming decision. 

e.g. if the default is `enabled`, and user wants to disable this, they would be able to do `--disable-backfill-rate-limtiing` instead of `--enable-backfill-rate-limiiting=false`, and the param doesn't have to take any value.

This is just my 2 cents, at the end of the day consistency is more important.

Reply:
I think you can safely declare a flag under that convention and in the code use a more readable version

Reply:
good idea, I'll go with that!

Reply:
Done!